shared/src/types/base/common.ts

```diff
@@ -1,7 +1,6 @@
 import { z } from 'zod'
 
-export const BaseModel = z.object({
-  id: z.number(),
+export const SequelizeTimestamps = z.object({
```

Comment:
nit - I generally favour the Scala approach of naming partial classes (traits) as `HasXXX`. Only a suggestion though.

Reply:
would this mean something like `HasTimestamps`? sorry not familiar with that convention

Reply:
Aye, something like `HasSequelizeTimestamps`, so that you get something like `HasSequelizeTimestamps.extend(...)`

Reply:
done in https://github.com/opengovsg/askgovsg/pull/298/commits/b0104045582b52da668bb3b72acd845f5491f82f